backend/src/main/java/com/bakdata/conquery/sql/models/ColumnDateRange.java

```diff
@@ -81,6 +81,18 @@ private ColumnDateRange as(String alias) {
 		);
 	}
 
+	public ColumnDateRange aggregated() {
+		if (isSingleColumnRange()) {
+			return of(DSL.field("range_agg({0})", this.range));
```

Comment:
ist das nicht nur in psql möglich?

Reply:
Genau, die singleColumnRange ist auch nur in Postgres möglich. Das ist nur ein Zwischenlösung bis zur Einführung von interval packing.